lib/renderer.js

```diff
@@ -892,6 +893,12 @@
                 inputRect = layer.bounds;
                 paddedBounds = inputRect;
             }
+        } else if (component.layerId === 0) { // whole doc
+            inputRect = component.document.bounds;
+            paddedBounds = component.document.bounds;
+
+            var contentBounds = this._getContentBounds(component.document.layers, false, component.document.bounds);
```

Comment:
For whole doc, why calculate ContentBounds? Isn't it just the doc bounds?

Reply:
Added code comments to explain

Reply:
We're getting contentBounds so we can be prepared for the state of the pixmap we will receive, right?

Reply:
correct